lib/macho/load_commands.rb

```diff
@@ -225,6 +282,30 @@ def to_i
         @string_offset
       end
     end
+
+    # Represents the contextual information needed by a load command to
+    # serialize itself correctly into a binary string.
+    class SerializationContext
+      # @return [Symbol] the endianness of the serialized string
+      attr_reader :endianness
+
+      # @return [Fixnum] the constant rounding value used to pad the serialized string
+      attr_reader :cmd_round
+
+      # @param macho [MachO::MachOFile] the file to contextualize
+      # @return [MachO::LoadCommand::SerializationContext] the resulting context
+      def self.context_for(macho)
```

Comment:
Thoughts on this versus a `MachOFile#lc_context` method? I couldn't decide between the two, so I just flipped a coin :smile: